arangod/Indexes/Index.cpp

```diff
@@ -415,6 +415,22 @@ bool Index::Compare(VPackSlice const& lhs, VPackSlice const& rhs) {
         return false;
       }
     }
+  } else if (type == IndexType::TRI_IDX_TYPE_IRESEARCH_LINK) {
+    value = lhs.get("view");
+    if (value.isString()) {
+      if (arangodb::basics::VelocyPackHelper::compare(
+              value, rhs.get("view"), false) != 0) {
+        auto ls = value.copyString();
+        auto rs = rhs.get("view").copyString();
+        if (ls.size() > rs.size()) {
+          std::swap(ls, rs);
+        }
+        if (ls.back() != '/' ||
+            ls.compare(rs.substr(0, ls.size())) != 0) {
```

Comment:
does this mean that `"view"` values of `"foobar"` and `"foo"` compare equal?

Reply:
or `"foo/"` and `"foo/baz"`?
Is that intentional?

Reply:
Sorry to ask silly questions, but I am not really aware of the context of this comparison and what values can be expected in the "view" attribute.